Wallet Download and Installation.md

```diff
@@ -0,0 +1,64 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page from the Skycoin Website to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## [Desktop Wallet](Desktop_Wallet)
+
+* ### [Windows](Windows)
+* ### [MAC OS](MAC_OS)
+* ### [Linux](Linux)
+
+## [Mobile Wallet](Mobile_Wallet)
+
+* ### [IOS](IOS)
+* ### [Android](Android)
+
+## <a name="Desktop_Wallet"> </a> Desktop Wallet
+
+### <a name="Windows"> </a> Windows
+
+You can follow these steps to install the Skycoin Desktop Wallet into your Windows PC:
+
+* Clicking on the link, "64 bit" will start downloading the installer for the Wallet.
+
+* Once the installer is downloaded, double click to open it.
+
+* The installation of the Desktop Wallet is similar to the installation of any other Windows software and after confirming the actions, the installation will be completed.
+
+* Once the installation is complete you can open the Wallet file from the location you had installed.
+
+### <a name="MAC_OS"> </a> MAC OS
+
+If you are using a MAC desktop then you can follow these steps when installing the Skycoin Desktop Wallet:
+
+* Download the .dmg file from the [Downloads](https://www.skycoin.net/downloads/) page.
+
+* Once you have downloaded the file double click to open it.
+
+* A window will open where you need to drag and drop the Desktop Wallet's icon to the "Applications" folder.
+
+* Once this is done, you can open the Wallet from the "Applications" folder.
+
+## <a name="Mobile_Wallet"> </a> Mobile Wallet
+
+### <a name="IOS"> </a> IOS
+
+To install the IOS version of the Skycoin Mobile Wallet into your iPhone, follow these steps:
+
+* Click on the link under IOS in the Skycoin [Downloads](https://www.skycoin.net/downloads/) page, from your iPhone.
+
+* The link will open the App Store application in your phone, where you will find the button "Get".
+
+* Clicking on "Get" will install the Wallet into your iPhone.
```

Comment:
Merge this  step with the previous one. Scrap the last part `where you will find the button get.`